include/neml2/models/Model.h

```diff
@@ -60,16 +61,39 @@ class Model : public NEML2Object,
   const LabeledAxis & output() const { return _output; }
   /// @}
 
+  void check_AD_limitation() const;
```

Comment:
At least add a docstring here to let us know what it doesn.

Reply:
Added